mods/sgssb/abilities.js

```diff
@@ -27,4 +27,24 @@ exports.BattleAbilities = {
 			this.setTerrain('');
 		},
 	},
+	conflictofinterest: {
+		id: "conflictofinterest",
+		name: "Conflict of Interest",
+		//Since levitate cant be coded in
```

Comment:
OK THE THRILLING CONCLUSION! I did some digging and so I found out there is this thingy in `/sim/pokemon.js`

`isGrounded(negateImmunity)`


Maybe we can use `isGrounded: false,` for what we can do?